resolwe_bio/tests/utils.py

```diff
@@ -1,380 +1,10 @@
-"""
-.. autoclass:: server.tests.processor.utils.PreparedData
-   :members:
+try:
+    from resolwe.flow.tests import ProcessTestCase
+except ImportError:
+    from .utils_genesis import ProcessTestCase
```

Comment:
I think you suggested this already, move utils_genesis to genesis. Tadej and I also agree on this. The import should then be:
```
from server.tests.utils import ProcessTestCase
# or
from server.tests.utils import ProcessorTestCase as ProcessTestCase
# or
from genesis.server.tests.utils import ProcessorTestCase as ProcessTestCase
```